phpseclib/Net/SFTP.php

```diff
@@ -1294,12 +1253,10 @@ public function lstat($filename)
      * Determines information without calling \phpseclib3\Net\SFTP::realpath().
      * The second parameter can be either PacketType::STAT or PacketType::LSTAT.
      *
-     * @param string $filename
-     * @param int $type
-     * @throws \UnexpectedValueException on receipt of unexpected packets
      * @return array|false
+     *@throws \UnexpectedValueException on receipt of unexpected packets
```

Comment:
I feel like there should be a space between `*` and `@throws`?

Reply:
Fixed all